ent/encryption/kms_host.cc

```diff
@@ -826,6 +828,11 @@ future<> encryption::kms_host::impl::init() {
     _initialized = true;
 }
 
+future<> encryption::kms_host::impl::stop() {
+    co_await _attr_cache.stop();
+    co_await _id_cache.stop();
```

Comment:
nit: any reason order is reversed compared to kmip_host::impl::disconnect?

Reply:
No. The order does not matter.

Reply:
I rrcommend to repeat the same order everywhere for symmetry reasons.